i18n/en.json

```diff
@@ -8,5 +8,12 @@
 	"wikibasereconcileedit-desc": "This is the ON OR Prototype extension",
 	"wikibasereconcileedit-editendpoint-invalid-reconcile-json": "Invalid reconcile JSON supplied",
 	"wikibasereconcileedit-editendpoint-unsupported-reconcile-version": "The only supported reconciliation {{PLURAL:$2|version is|versions are}} $1",
-	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123"
+	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123",
+	"wikibasereconcileedit-editendpoint-invalid-property-type-must-be-url": "$1 property $2 must be of type url. The datatype of the provided property is $3",
+	"wikibasereconcileedit-editendpoint-unspecified-entity-input-version": "Entity input version must be specified",
+	"wikibasereconcileedit-editendpoint-invalid-entity-input-version": "Unknown entity input version $1; the only supported {{PLURAL:$3|version is|versions are}} $2",
+	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
+	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
+	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must contain a value",
+	"wikibasereconcileedit-editendpoint-matched-multiple-items": "Matched $2 Items during reconciliation :(\n\n{{PLURAL:$2|item is|items are}} $1\n"
```

Comment:
Hang on, this message isn’t actually used (probably because the corresponding `die()` was moved to `ReconciliationService` in the meantime). I guess we should remove the last commit for now?

Reply:
:(